v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/options/OptionsToConfigBuilder.java

```diff
@@ -204,5 +201,27 @@ protected static String addParamToJdbcUrl(String jdbcUrl, String paramName, Stri
     }
   }
 
+  private static String extractDbFromURL(String sourceDbUrl) {
+    URI uri;
+    try {
+      // Strip off the prefix 'jdbc:' which the library cannot handle.
+      uri = new URI(sourceDbUrl.substring(5));
+    } catch (URISyntaxException e) {
+      throw new RuntimeException(String.format("Unable to parse url: %s", sourceDbUrl), e);
+    }
+    // Remove '/' before returning.
+    return uri.getPath().substring(1);
+  }
+
+  private static JdbcIOWrapperConfig.Builder builderWithDefaultsFor(SQLDialect dialect) {
+    switch (dialect) {
+      case MYSQL:
+        return builderWithMySqlDefaults();
+      case POSTGRESQL:
+        return builderWithPostgreSQLDefaults();
+    }
+    return null;
```

Comment:
[nit] From a POV of code coverage and static null checkers, would it make sense to add handle MySQL as default case and avoid the un-reachable null return?

Reply:
Updated